airbyte-integrations/connectors/source-pokeapi/source_pokeapi/schemas/pokemon.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "name": {
```

Comment:
if I run `curl https://pokeapi.co/api/v2/pokemon/pikachu` the schema looks way more complicated. This should include it. This allows downstream consumers to know how to format/normalize the data.

Reply:
Yep makes sense, done!